Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/storage/EventDatabase.kt

```diff
@@ -0,0 +1,35 @@
+package de.rki.coronawarnapp.eventregistration.checkins.storage
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import de.rki.coronawarnapp.eventregistration.checkins.storage.dao.EventCheckInDao
+import de.rki.coronawarnapp.eventregistration.checkins.storage.entity.EventCheckInEntity
+import de.rki.coronawarnapp.util.database.CommonConverters
+import de.rki.coronawarnapp.util.di.AppContext
+import javax.inject.Inject
+
+@Database(
+    entities = [
+        EventCheckInEntity::class
+    ],
+    version = 1,
+    exportSchema = true
+)
+@TypeConverters(CommonConverters::class)
+abstract class EventDatabase : RoomDatabase() {
```

Comment:
shouldn't be CheckInDatabase?

Reply:
i thought we could use one db for all the event stuff?

Reply:
Please align with @LukasLechnerDev, I think he also created a database

Reply:
we aligned.

Reply:
I will also use this database. So we have to first merge this PR and then I can adapt mine.